### DISCUSSION TITLE
My Shelly Mini1G3 responds slowly

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#21631] [Q&A]
I have installed my Shelly behind a light switch, but have noticed that the wif sometimes reacts very slowly.

I have an RSSI of 34%, -83 dBm, sometimes only 30%.

Is this due to this value? If so, how can I counteract this?

**Comment by sfromis** [comment]
Signal strength of -83 dBM is **horrible**, no reliable communication feasible. Fixing such very bad wifi is about looking at factors working against radio propagation, both distance, blocking factors (like metal or heavy concrete), or other RF sources on the 2.4 GHz band.
